servers/rendering/renderer_rd/renderer_canvas_render_rd.h

```diff
@@ -358,15 +381,20 @@ class RendererCanvasRenderRD : public RendererCanvasRender {
 				uint32_t colors[6]; // colors encoded as half
 			};
 		};
-		float color_texture_pixel_size[2];
+		uint32_t texture_data_index;
+		uint32_t pad;
```

Comment:
@clayjohn / @huwpascoe this is the freed up slot in the `InstanceData` buffer that can be used for #97058